bin/backup.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+#######################################
+#  Perform a full backup of the site
+#  on the users home.
+#  This script can be called directly
+#  or automated by cron, as it do a
+#  cleanup for oldest backups.
+#
+#             WARNING!
+#  Execute from the project root
+#######################################
+
+set -x
```

Comment:
This is the debug flag. I would use -e instead, so it stops on error. Debug can be enabled from the command line if needed.